pkg/extensions/search/schema.graphql

```diff
@@ -124,70 +124,100 @@ type ImageSummary {
     Timestamp of the last modification done to the image (from config or the last updated layer)
     """
     LastUpdated: Time
     """
+    Human-readable description of the software packaged in the image
+    """
+    Description: String
+    """
     True if the image has a signature associated with it, false otherwise
     """
-    IsSigned: Boolean
+    IsSigned: Boolean 
     """
-    Total size of the files associated with this image (manigest, config, layers)
+    License(s) under which contained software is distributed as an SPDX License Expression
     """
-    Size: String
+    Licenses: String  #  The value of the annotation if present, 'unknown' otherwise).
     """
-    OS and architecture supported by this image
+    Labels associated with this image
+    NOTE: currently this field is unused
     """
-    Platform: OsArch
+    Labels: String
     """
-    Vendor associated with this image, the distributing entity, organization or individual
+    Human-readable title of the image
     """
-    Vendor: String
+    Title: String
     """
     Integer used to rank search results by relevance
     """
     Score: Int
     """
-    Number of downloads of the manifest of this image
+    URL to get source code for building the image
     """
-    DownloadCount: Int
+    Source: String
     """
-    Information on the layers of this image
+    URL to get documentation on the image
     """
-    Layers: [LayerSummary]
+    Documentation: String
     """
-    Human-readable description of the software packaged in the image
+    Vendor associated with this image, the distributing entity, organization or individual
     """
-    Description: String
+    Vendor: String
     """
-    License(s) under which contained software is distributed as an SPDX License Expression
+    Contact details of the people or organization responsible for the image
     """
-    Licenses: String
+    Authors: String
     """
-    Labels associated with this image
-    NOTE: currently this field is unused
+    Short summary of the identified CVEs
     """
-    Labels: String
+    Vulnerabilities: ImageVulnerabilitySummary
+}
+"""
+Details about a specific version of an image for a certain operating system and architecture.
+"""
+type ManifestSummary {
     """
-    Human-readable title of the image
+    Digest of the manifest file associated with this image
     """
-    Title: String
+    Digest: String
     """
-    URL to get source code for building the image
+    Digest of the config file associated with this image
     """
-    Source: String
+    ConfigDigest: String
     """
-    URL to get documentation on the image
+    Timestamp of the last update to an image inside this repository
     """
-    Documentation: String
+    LastUpdated: Time
+    """
+    Total size of the files associated with this manifest (manigest, config, layers)
+    """
+    Size: String
+    """
+    OS and architecture supported by this image
+    """
+    Platform: OsArch
```

Comment:
OsArch -> Platform

Reply:
@rchincha are we planning to change the API here?